packages/cosmic-swingset/src/chain-main.js

```diff
@@ -208,6 +208,10 @@ export default async function main(progname, args, { env, homedir, agcc }) {
       },
       exportMailbox,
     );
+    function setCrankHash(crankhash) {
+      const msg = stringify({ method: 'set', key: `crankhash`, crankhash });
```

Comment:
I missed this on the first time through:

```suggestion
      const msg = stringify({ method: 'set', key: 'crankhash', value: crankhash });
```